src/libraries/System.Private.CoreLib/src/System/Enum.cs

```diff
@@ -1086,6 +1275,52 @@ public static string Format(Type enumType, object value, [StringSyntax(StringSyn
 
             throw new FormatException(SR.Format_InvalidEnumFormatSpecification);
         }
+
+        public static bool TryFormat<TEnum>(TEnum value, Span<char> destination, out int charsWritten, [StringSyntax(StringSyntaxAttribute.EnumFormat)] ReadOnlySpan<char> format) where TEnum : struct, Enum
+        {
+            if (!destination.IsEmpty && format.Length == 1)
+            {
+                // Optimize for these standard formats that are not affected by culture.
+                switch ((char)(format[0] | 0x20))
```

Comment:
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYQYG8aHunHjykDMAAtsUBgA0AFACUY2ACYB5AHYAbAJ4BlAA7YVAHhFiAfAwBm0fNgwBKDlx5PiAdgZTjUW1MtRrGAG0ABgBdBkIGIIRSINsAbkduAF8aJKA==

@EgorBo Why are we making a modreq call here (see IL)? Can't something like this be constantly evaluated?

Reply:
That `call` you see is the `ReadOnlySpan`'s accessor. The `modreq([System.Runtime]System.Runtime.InteropServices.InAttribute)` means that the return type is a `readonly ref`.

Reply:
okay. Why isn't it inlined?